server/database/dbSeed.js

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
+const mocker = require('mocker-data-generator').default;
+const db = require('./index.js');
+
+// mock data template for review data
+const reviewsTemplate = {
+  url_id: {
+    faker: 'random.number({"min": 0, "max": 99})',
+  },
+  name: {
+    faker: 'name.firstName',
+  },
+  location: {
+    faker: 'address.state',
+  },
+  date: {
+    faker: 'date.past',
+  },
+  comment: {
+    faker: 'lorem.paragraph',
+  },
+  rating: {
+    faker: 'random.number({"min": 1, "max": 5})',
+  },
+};
+
+const databaseInsertion = (data) => {
+  const queryString = 'INSERT INTO reviews (url_id, name, location, date, comment, rating) VALUES (?, ?, ?, ?, ?, ?)';
+  db.query(queryString, data, (err, results) => {
+    if (err) {
+      return err;
+    }
+    return results;
+  });
```

Comment:
@Matt-Wigg, have you tested this code? It will likely work, given that resolving all promises is doing only console.log('** reviews database seeded **'));

However, you may want to ask yourself whether this is the correct approach, and what will happen if you tried to access the results from Promise.all().then()?

I see an anonymous callback being passed to db.query call and returning the results. Where do those results end up being? Please feel free to zoom in and we can discuss this together.